system-test/fixtures/sample/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "./node_modules/gts/tsconfig-google.json",
+  "compilerOptions": {
+    "rootDir": ".",
+    "outDir": "build"
```

Comment:
Should we try setting `types: [ "node" ]` here to see what happens?  Maybe to repair https://github.com/googleapis/nodejs-pubsub/issues/526?

Reply:
Also, skipLibCheck?  https://github.com/googleapis/nodejs-pubsub/issues/532

Reply:
SGTM! Should we consider testing against yarn pnp too?

Reply:
Eh, other priorities right now :) If it's easy, go fer it